client/app/services/usage-graphs/usage-graphs.html

```diff
@@ -0,0 +1,76 @@
+  <h2 translate>Utilization</h2>
+  <hr>
+  <div class="row">
+    <div class="col-md-4 text-center">
+                <div>
+                    <div class="metric-title" translate>CPU</div>
+                    <div class="metric-details-container">
+                        <div ng-if="vm.cpuDataExists">
+                            <span class="metric-number">{{vm.availableCPU}}</span>
+                            <span class="metric-details">
+                                <p translate>mhz available</p>
+                                <p><span translate>of</span> {{vm.cpuChart.data.total}} <span translate>mhz</span></p>
+                            </span>
+                        </div>
+                    </div>
+                    <hr>
+                    <div ng-if="!vm.cpuDataExists">
+                      <div>
+                          <i class="pf pficon-info"></i>
+                      </div>
+                      <div translate>
+                        No data available
+                      </div>
+                    </div>
+                </div>
+                <div pf-donut-pct-chart ng-if="vm.cpuDataExists" config="vm.cpuChart.config" data="vm.cpuChart.data" center-label="vm.cpuChart.label"></div>
+    </div>
+    <div class="col-md-4 text-center">
+        <div>
+            <div class="metric-title" translate>Memory</div>
+            <div class="metric-details-container">
+                <div ng-if="vm.memoryDataExists">
+                    <span class="metric-number">{{vm.availableMemory}}</span>
+                    <span class="metric-details">
+                        <p translate>gb available</p>
```

Comment:
Need to follow PatternFly standards.  "GB" should be in all caps.  "Available" should have a capital "A".